tasks/vscode.py

```diff
@@ -131,3 +131,35 @@ def setup_extensions(ctx: Context):
     for extension in content.get("recommendations", []):
         print(color_message(f"Installing extension {extension}", Color.BLUE))
         ctx.run(f"code --install-extension {extension} --force")
+
+
+@task
+def setup_tests(_, force=False):
+    """
+    Setup the tests tab for vscode
+
+    - Documentation: https://datadoghq.atlassian.net/wiki/x/z4Jf6
+    """
+    from invoke_unit_tests import TEST_ENV
+
+    if not force and os.path.exists('.env'):
+        raise Exit(color_message('The .env file already exists', Color.RED))
+
+    with open('.env', 'w') as f:
+        for key, value in TEST_ENV.items():
+            print(f'{key}={value}', file=f)
+
+    print(color_message('The .env file has been created', Color.GREEN))
+
+    print(
+        f'''{color_message('note', Color.BLUE)}: You should add the following to your .vscode/settings.json file:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What if we did this when setting up the settings file? I think there is a task that was recently added to create the vs code settings file

Reply:
Maybe not, could not find it

Reply:
Ah here it is, it is open https://github.com/DataDog/datadog-agent/pull/27329 but could use this adds-on

Reply:
Good idea, I'll do it when this gets merged